content-src/asrouter/docs/targeting-attributes.md

```diff
@@ -566,3 +567,20 @@ declare const userPrefs: {
   snippets: boolean;
 }
 ```
+
+### `listAttachedOAuthClients`
+
+Information about connected services associated with the FxA Account.
```

Comment:
Might be worth adding an example of available clients (or actually add them to the type info, since users will need that)